src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -10847,26 +10853,28 @@ private void Foo()
 
     void Bar2(Action t) { }
 }";
-            var expected = @"
+            var expected = $@"
 using System;
 class Program
-{
+{{
     private void Bar()
-    {
-        Bar2(Foo;);
-    }
+    {{
+        Bar2(Foo{commitChar});
```

Comment:
This is where things get interesting. `;` should trigger Complete Statement and produce `Bar2(Foo);$$;`, while `.` should insert parentheses to give `Bar2(Foo().$$);`.

Reply:
Same as above, this is a good corner case to add tests

Reply:
This one is different because one adds parentheses but the other doesn't.